odsclient/core.py

```diff
@@ -430,7 +501,7 @@ def _http_call(self,
                 # }
                 details = loads(body)
             except JSONDecodeError:
-                # error parsing the json payload?
+                # error parsing the json dataset?
```

Comment:
this seems like a replacement mistake

```suggestion
                # error parsing the json payload?
```